logging/zerolog_adapter/logger.go

```diff
@@ -72,43 +72,83 @@ func getLogWriter(logFileName string) (io.Writer, error) {
 }
 
 func (l Logger) Debug(args ...interface{}) {
-	l.Logger.Debug().Timestamp().Msg(fmt.Sprint(args...))
+	event := l.Logger.Debug()
+	if event == nil {
+		return
+	}
```

Comment:
Which circumstance can lead to event to be nil?

Reply:
zerolog returns `event = nil` when log level is disabled, and do nothing on all next calls `event.Fileds(...).Timestamp().Msg(...)`